opencensus/OpenCensus-Metrics.md

```diff
@@ -218,6 +244,22 @@ These metrics represent the total sum of all values recorded since recording beg
 
 These metrics must be translated to New Relic's Count metric type. To record them, take the current value of the SumData metric and subtract from it the previous value of the SumData metric. If this is the first time this metric is seen, use its total value. All delta values greater than or equal to zero must be sent.
 
+#### Queries
+
+To see a line chart detailing the sum of all the values of `MySum` over each timeslice use this NRQL query:
+
+```sql
+NRQL> SELECT sum(MySum) FROM Metric TIMESERIES AUTO
```

Comment:
isn't `auto` the default on a `timeseries`? If so, we can probably get rid of it in these examples.

Reply:
Oh good call, I didn't realize auto was auto 😆